performance_regression_test.py

```diff
@@ -330,7 +330,7 @@ def run_workload(self, stress_cmd, nemesis=False, sub_type=None):
         stress_queue = self.run_stress_thread(stress_cmd=stress_cmd, stress_num=1, stats_aggregate_cmds=False)
         if nemesis:
             interval = self.params.get('nemesis_interval')
-            time.sleep(interval * 60)  # Sleeping one interval (in minutes) before starting the nemesis
+            time.sleep(5 * 60)  # Sleeping one interval (in minutes) before starting the nemesis
```

Comment:
This is just for testing ? right ?

Reply:
I try to make it working ASAP, so I took some shortcuts here and there. Later I'll remove them.